drivers/serial/uart_nrfx_uarte.c

```diff
@@ -776,9 +782,12 @@ static void rxto_isr(struct device *dev)
 		user_callback(dev, &evt);
 		data->async->rx_next_buf = NULL;
 	}
-	evt.type = UART_RX_DISABLED;
 
-	user_callback(dev, &evt);
+	/* Flushing RX fifo requires buffer bigger than 4 bytes to empty fifo */
+	static u8_t flush_buf[5];
```

Comment:
static. Isn't there more instances? Are those bytes discarded anyway?

Reply:
There are more instances but bytes are discarded anyway, thats why single buffer for all instances is used.